src/main/msp/msp.c

```diff
@@ -4051,10 +4050,25 @@ static mspResult_e mspProcessInCommand(mspDescriptor_t srcDesc, int16_t cmdMSP,
                     textVar = currentControlRateProfile->profileName;
                     break;
 
+                case MSP2TEXT_CUSTOM_MSG_0:
+                case MSP2TEXT_CUSTOM_MSG_0 + 1:
+                case MSP2TEXT_CUSTOM_MSG_0 + 2:
+                case MSP2TEXT_CUSTOM_MSG_0 + 3:
+                    {
+                        unsigned msgIdx = textType - MSP2TEXT_CUSTOM_MSG_0;
```

Comment:
Does this need changing if `OSD_CUSTOM_MSG_COUNT` is changed in future. If so this deserves comment.

Reply:
Only if `CUSTOM_MSG_MAX_NUM` is changed; changing CUSTOM_MSG_MAX_NUM will break MSP protocol IDs that are added after MSP2TEXT_CUSTOM_MSG, so unlikely.
It was originally 8 (allocating 8 IDs for custom text, but without allocating memory), but that was changed after your review.